addons/block_code/ui/blocks/utilities/template_editor/template_editor.gd

```diff
@@ -87,26 +88,36 @@ func _update_from_format_string():
 		child.queue_free()
 
 	var match_id = 0
+	var is_collapsable = false
+	var collapsable := CollapsableSettingsScene.instantiate() as CollapsableSettings
 	for item in BlockDefinition.parse_display_template(format_string):
+		var hidden = item.has("hidden") and item.get("hidden")
```

Comment:
`item.get("hidden")` already returns `null` if the key is missing, and you can also pass a default. So you can probably do the same without the `has()` check:
```suggestion
		var hidden = item.get("hidden", false)
```
Untested, please confirm!